src/content/app-platform/create-catalog/index.md

```diff
@@ -0,0 +1,161 @@
+linkTitle: Creating an app catalog
+title: Creating an app catalog
+description: How to create a custom app catalog for use with app platform and push helm charts to it.
+weight: 40
+menu:
+  main:
+    parent: app-platform
+last_review_date: 2022-01-25
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-honeybadger
+user_questions:
+  - How can I create an app catalog for my apps?
+  - How can I serve an app catalog using GitHub Pages?
+  - How can I publish an app to my app catalog?
+  - How can I create a catalog CR for my app catalog?
+  - How can I create an app CR for an app in the catalog?
+---
+
+# Creating an app catalog
+
+## Overview
+
+An app catalog is a collection of apps that can be deployed using the Giant Swarm App Platform.
+We extend Helm and an app catalog is also a Helm [Chart Repository](https://helm.sh/docs/topics/chart_repository/).
+Each app catalog has its own [Catalog]({{< relref "/ui-api/management-api/crd/catalogs.application.giantswarm.io.md" >}})
+CR in the management cluster.
+
+The app catalog is a Git repository that contains a Helm `index.yaml` and chart
+tarballs for each version of the app that has been published. These files must
+be served over HTTP.
+
+We recommend using our [app-build-suite](https://github.com/giantswarm/app-build-suite/)
+tool to publish apps to your catalog. It adds additional metadata files that
+allows app platform to extend Helm. Such as only allowing an app to be
+installed once in a cluster. You can learn more about the tool by reading its
+[tutorial](https://github.com/giantswarm/app-build-suite/blob/master/docs/tutorial.md).
+
+These metadata files and the Helm `index.yaml` are used to generate app metadata
+that is stored in the management cluster in [App Catalog Entry]({{< relref "/ui-api/management-api/crd/appcatalogentries.application.giantswarm.io.md" >}})
+CRs.
+
+There are [multiple options](https://helm.sh/docs/topics/chart_repository/#hosting-chart-repositories)
+for serving the catalog over HTTP including GitHub Pages or tools like Harbor or ChartMuseum which run in a
+Kubernetes cluster. At Giant Swarm we use GitHub Pages and this is what we will
+cover in this guide.
+
+## Creating an app catalog hosted using GitHub Pages
+
+First, you should choose a name for your catalog. We recommend using the suffix
+`-catalog` to make it clear this Git repository hosts an app catalog.
+
+Create the Git repository in GitHub as a public repository and enable
+[GitHub Pages](https://docs.github.com/en/pages/quickstart) for the `main` branch.
+
+## Publishing an app to the app catalog
+
+Now you can configure your apps to be published to your catalog. We have created
+a GitHub Action for `app-build-suite` that will automate this.
+
+```yaml
+# .github/workflows/push-to-app-catalog.yaml
+name: 'Push to App Catalog'
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  push_to_app_catalog:
+    uses: giantswarm/app-build-suite/.github/workflows/push-to-app-catalog.yaml@master
```

Comment:
How should we version the action?

I think we should use tagged releases and recommend customers set up dependabot to keep the action updated. WDYT?

https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/keeping-your-actions-up-to-date-with-dependabot